src/ui/components/filters/rangefiltercomponent.js

```diff
@@ -7,6 +7,9 @@ import FilterNodeFactory from '../../../core/filters/filternodefactory';
 import FilterMetadata from '../../../core/filters/filtermetadata';
 import ComponentTypes from '../../components/componenttypes';
 import TranslationFlagger from '../../i18n/translationflagger';
+import StorageKeys from '../../../core/storage/storagekeys';
```

Comment:
can we group the imports together have similar paths? e.g. move the core imports up next to FilterMetadata?